OutOfSchool/OutOfSchool.Encryption/Services/DevEUSignOAuth2Service.cs

```diff
@@ -0,0 +1,19 @@
+using System.Text.Json;
+using OutOfSchool.Encryption.Models;
+
+namespace OutOfSchool.Encryption.Services;
+
+public class DevEUSignOAuth2Service : IEUSignOAuth2Service
+{
+    /// <inheritdoc/>
+    public CertificateResponse GetEnvelopCertificateBase64() => new()
+    {
+        CertBase64 = Convert.ToBase64String("very_secret_certificate"u8.ToArray()),
```

Comment:
should this name be configurable ?

Reply:
@h4wk13 dev mock authorization server has also this hardcoded as it does not implement any encryption.